flight_phase_files/RF07_info.yaml

```diff
@@ -47,17 +47,21 @@
     start: 2020-01-31 18:52:58
     end: 2020-01-31 19:06:16
   - kind: radar_cal_tilted
-    name: radar_cal_tilted
+    name: radar_cal_tilted (10deg) around Ron Brown
```

Comment:
Remove additional comment after "radar_cal_tilted". "tilted" is sufficiently specific here. Additional information is only required for the straight legs for now, e.g. "straight_leg_towards_ron_brown".